app/services/organization/organization.service.go

```diff
@@ -0,0 +1,92 @@
+package user
+
+import (
+	"context"
+	model "database-ms/app/models"
+	"database-ms/config"
+	"database-ms/utils"
+
+	"github.com/google/uuid"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type OrganizationServiceInterface interface {
+	Create(context.Context, *model.Organization) error
+	// FindByThingId(context.Context, string) ([]*model.Organization, error)
+	Update(context.Context, string, *model.Organization) error
+	Delete(context.Context, string) error
+	GetOrganizations(context.Context, string) ([]*model.Organization, error)
+}
+
+type OrganizationService struct {
+	db     *mgo.Session
+	config *config.Configuration
+}
+
+func NewOrganizationService(db *mgo.Session, c *config.Configuration) OrganizationServiceInterface {
+	return &OrganizationService{config: c, db: db}
+}
+
+func (service *OrganizationService) Create(ctx context.Context, organization *model.Organization) error {
+	organization.ID = bson.NewObjectId()
```

Comment:
It might be good to clarify with Justin T if we need to send the organization's id when we create them. If the object id is created here, then it will be difficult to pass it back as a response

Reply:
Sounds good!